projects/mno-frontend-lib/src/lib/_components/login-box/login-box.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { AuthenticationService } from '../../_services/authentication.service';
+import { User } from '../../_models/user';
+
+@Component({
+  selector: 'mno-login-box',
+  templateUrl: './login-box.component.html',
+  styleUrls: ['./login-box.component.css']
+})
+export class LoginBoxComponent implements OnInit {
+  @Input() header: string;
+  @Output() onLogin = new EventEmitter<User>();
```

Comment:
Output naming convention for angular is to NOT use the 'on' prefix (see https://angular.io/guide/styleguide#dont-prefix-output-properties).

"onLogin" => "loggedIn"
"onCancel" => "cancelled"

@cesar-tonnoir use a linter :)